site-src/concepts/versioning.md

```diff
@@ -1,116 +1,70 @@
-  should implement their own escaping or validation as necessary. To avoid
-  duplicating work, Gateway API maintainers are considering adding a shared
-  validation package that implementations can use for this purpose. This is
-  tracked by [#926](https://github.com/kubernetes-sigs/gateway-api/issues/926).
-
-### Installers:
-* MUST install a full Gateway API bundle, with matching CRD and webhook
-  versions.
-
-## Adding Experimental Fields
-Over time, it will be useful to add experimental fields to the API. In upstream
-Kubernetes, those would generally be guarded with feature gates. With Gateway
-API we will accomplish by releasing experimental versions of our CRDs.
-
-With this approach, we achieve a similar result. Instead of using feature gates
-and validation to prevent fields from being set, we just release separate CRDs.
-Once the API reaches beta, each bundle release can include 2 sets of CRDs,
-stable and experimental.
-
-New fields will be added to the experimental set of CRDs first, and may graduate
-to stable APIs later. Experimental fields will be marked with the
-`+experimental` annotation in Go type definitions. Gateway API CRD generation
-will exclude these fields from stable CRDs. Experimental fields may be removed
-from the API. Due to the experimental nature of these CRDs, they are not
-recommended for production use.
-
-If experimental fields are removed or renamed, the original field name should be
-removed from the go struct, with a tombstone comment ensuring the field name
-will not be reused. 
-
+### API Versions (v1alpha2, v1beta1)
+Within Gateway API, API versions are primarily used to indicate the stability of
+a resource. For example, if a resource has not yet graduated to beta, it is
+still possible that it could either be removed from the API or changed in
+backwards incompatible ways. For more information on API versions, refer to the
+[full Kubernetes API versioning
+documentation](https://kubernetes.io/docs/reference/using-api/#api-versioning).
+
+### Release Channels (experimental, stable)
+Release channels are used to indicate the stability of fields within a resource.
+For example, even if a resource has already graduated to beta, we need a way to
+test out new fields and features before committing to their long term stability.
+With upstream Kubernetes APIs, this is accomplished with [feature
+gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
+Unfortunately, CRDs do not have a similar concept yet.
+
+Instead of trying to recreate feature gates in this project, we've introduced
+release channels. Starting in v0.5.0, all new fields and features will start in
+the experimental release channel. From that point, these may graduate to the
+stable release channel or be dropped from the API entirely.
+
+Due to their unstable nature, experimental CRDs should not be used for
+production workloads.
+
```

Comment:
Rob -- I think  we need an example to illustrate the differences between version, channel?

What do you think? I think it will make it very clear for the reader what is intended for this.

Reply:
I think I went overboard here, but hopefully the diagrams here help: https://deploy-preview-1066--kubernetes-sigs-gateway-api.netlify.app/concepts/versioning/